fe/projects/AspNetCore/src/shared/components/NotificationBar/NotificationBar.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as Icon from "@mui/icons-material";
+import { Splide, SplideSlide } from "@splidejs/react-splide";
+
+const NotificationBar = (props) => {
+
+    const getIconTag = (iconName) => {
+        const IconTag = Icon[iconName];
+
+        if (IconTag === undefined || IconTag === null) {
+            IconTag = Icon["PushPin"];
+        }
+
+        return (
+            <span>
+                <IconTag />
+            </span>
+        )
+    };
+
+    return (
+        <div className="notification-bar">
+            <div className="notification-bar_desktop">
+                <div className="notification-bar_list is-flex is-justify-content-center">
+                    {props.items.map((item) => (
+                        <div className="notification-bar_item is-flex">
+                            {getIconTag(item.icon)}
+                            <a href={item.link.url} target={item.link.target} className="notification-bar_link">
+                                {item.link.text}
+                            </a>
+                        </div>
+                    ))}
+                </div>
+            </div>
+            <div className="notification-bar_mobile">
```

Comment:
**suggestion:** When is only one element in items. Displayed:

![obraz](https://github.com/dawiddworak88/Giuru/assets/93117743/8e6dbbd3-c794-4c7f-a122-9a2e5c0744ad)